doc/examples/opencv/README.md

```diff
@@ -2,17 +2,12 @@
 
 ## Intro
 
-In this tutorial we're going to do edge detection using the Canny edge
-detection algorithm implemented in OpenCV. We'll be deploying our code as a
-Pachyderm pipeline which means it will be both streaming and distributed.
+In this tutorial we're going to do edge detection using the Canny edge detection algorithm implemented in OpenCV. We'll be deploying our code as a Pachyderm pipeline which means it will be both streaming and distributed.
```

Comment:
Do we want to replace with guide with the beginner tutorial at some point? Not worth it now I guess and I like that we have them build the docker image, but the rest is better in the tutorial IMO since that was recently rewritten. We can earmark this for later